src/lex.js

```diff
@@ -1115,7 +1115,6 @@ Lexer.prototype = {
       char = "\\\\";
       break;
     case "\"":
-      char = "\\\"";
```

Comment:
Since this "case" clause no longer has any purpose, we can remove it entirely.

Reply:
Okay will do that and will also check for any breaking changes after that.

Reply:
Done, I have made the changes suggested by you in the commit `
cf99223 `.